src/controllers/orderStatus.controller.js

```diff
@@ -1,7 +1,6 @@
-import { Sales } from '../database/models';
-import { Orders } from '../database/models';
+import { Sales, Orders } from '../database/models';
 
-const trackOrderStatus = async (req, res) => {
+export const trackOrderStatus = async (req, res) => {
   try {
     const buyerId = req.user.id;
     const selectedOrder = await Orders.findOne({
```

Comment:
This should not be here

Reply:
@ba-lambert  Fixed

Reply:
What I mean is that you have to respect folder structuring